example/example.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:yaml/yaml.dart';
+
+// ignore: always_declare_return_types
```

Comment:
Instead of having the ignore in the example, just add a return type! `void~

Reply:
Done!